src/App.tsx

```diff
@@ -10,9 +10,10 @@ import SignInPage from "./Pages/SignIn";
 import SignUpPage from "./Pages/SignUp";
 import StartPage from "./Pages/StartPage";
 import { createTheme, colors, ThemeProvider, Typography } from "@mui/material";
-import AuthProvider from "./Contexts/AuthContext";
+import { AuthProvider } from "./Contexts/AuthContext";
```

Comment:
Why the curly braces here and not around UserProvider?

Reply:
I blame prettier for that :3